packages/@react-aria/datepicker/src/useDatePicker.ts

```diff
@@ -61,31 +81,35 @@ export function useDatePicker(props: DatePickerAriaProps, state: DatePickerState
   };
 
   return {
-    comboboxProps: mergeProps(domProps, {
-      role: 'combobox',
-      'aria-haspopup': 'dialog',
-      'aria-expanded': state.isOpen,
-      'aria-owns': state.isOpen ? dialogId : null,
-      'aria-invalid': state.validationState === 'invalid' || null,
-      'aria-errormessage': props['aria-errormessage'],
+    groupProps: mergeProps(domProps, {
+      role: 'group',
+      'aria-describedby': dateValueDescription ? descId : null,
       'aria-disabled': props.isDisabled || null,
-      'aria-readonly': props.isReadOnly || null,
-      'aria-required': props.isRequired || null,
       ...mergeProps(pressProps, {onKeyDown}),
       ...labels
     }),
+    descProps: {
+      children: dateValueDescription,
+      hidden: true,
+      id: descId
+    },
     fieldProps: {
+      role: 'presentation',
+      'aria-controls': state.isOpen ? dialogId : null,
+      'aria-haspopup': 'dialog',
+      'aria-invalid': state.validationState === 'invalid' || null,
```

Comment:
Curious why you added these? I think `aria-haspopup` and `aria-invalid` aren't allowed on `role="presentation"` at least in ARIA 1.2. Did these have an effect on announcements anywhere?